salt/roster/flat.py

```diff
@@ -44,19 +44,20 @@ def targets(tgt, tgt_type='glob', **kwargs):
     conditioned_raw = {}
     for minion in raw:
         conditioned_raw[str(minion)] = raw[minion]
-    rmatcher = RosterMatcher(conditioned_raw, tgt, tgt_type, 'ipv4')
+    rmatcher = RosterMatcher(conditioned_raw, tgt, tgt_type, 'ipv4', __opts__)
     return rmatcher.targets()
 
 
 class RosterMatcher(object):
     '''
     Matcher for the roster data structure
     '''
-    def __init__(self, raw, tgt, tgt_type, ipv='ipv4'):
+    def __init__(self, raw, tgt, tgt_type, ipv='ipv4', opts={}):
```

Comment:
Ok.

Can you please you are sure about this. The code above has also kwargs, but here we are talking about passing global `__opts___` to avoid passing it as a global. If I use `kwargs` where you indicate, I would end merging `__opts__` into the function `kwargs`, which I am not sure it is better.